test/suites/dev/session-keys/test_session_keys.ts

```diff
@@ -33,7 +33,6 @@ describeSuite({
             // Check authorities are correct
             const authorities = (await polkadotJs.query.aura.authorities());
             expect(u8aToHex(authorities[0])).to.be.eq(u8aToHex(alice.publicKey));
```

Comment:
I recently made a PR to use to.deep.equal in cases like these, did I miss this line or did you forget to merge master?

```suggestion
            expect(authorities.toJSON()).to.deep.equal([u8aToHex(alice.publicKey)]);
```